.github/workflows/build.yaml

```diff
@@ -0,0 +1,105 @@
+name: Build
+
+# Run manually or each time a tag is pushed
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+
+# Enable stack protection
+env:
+  LDFLAGS: -fstack-protector
+
+jobs:
+  build:
+    name: "Build Chimera"
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: "Install deps"
+        run: |-
+          sudo apt-get update
+          sudo apt-get install -y software-properties-common gnupg build-essential cmake python3 p7zip-full curl
+
+          # Add debian testing to get mingw-w64 v8.0.0 (gcc/g++ v10.1.0)
+          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC
+          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 648ACFD622F3D138
+          sudo add-apt-repository "deb http://ftp.de.debian.org/debian sid main"
+          sudo apt-get install -t sid -y mingw-w64
+
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Compile"
+        run: |-
+          # Make CMake toolchain file
+          printf '%s\n' \
+            'set(CMAKE_SYSTEM_NAME Windows)' \
+            'set(TOOLCHAIN i686-w64-mingw32)' \
+            'set(CMAKE_C_COMPILER ${TOOLCHAIN}-gcc-posix)' \
+            'set(CMAKE_CXX_COMPILER ${TOOLCHAIN}-g++-posix)' \
+            'set(CMAKE_RC_COMPILER ${TOOLCHAIN}-windres)' \
+            'set(CMAKE_AR ${TOOLCHAIN}-gcc-ar-posix)' \
+            'set(CMAKE_RANLIB ${TOOLCHAIN}-gcc-ranlib-posix)' \
+            'set(CMAKE_FIND_ROOT_PATH /usr/${TOOLCHAIN})' \
+            'set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)' \
+            'set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)' \
+            'set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)' \
+            'set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)' \
+          > toolchain.cmake
+
+          cmake . -B build \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DCMAKE_TOOLCHAIN_FILE=toolchain.cmake \
+            -DCMAKE_INSTALL_PREFIX=build
+
+          cd build
+          make -j $(nproc)
+
+      - name: "Get fonts"
+        run: |-
+          mkdir fonts
+          cd fonts
+          curl -sL 'https://github.com/source-foundry/Hack/releases/download/v3.003/Hack-v3.003-ttf.tar.gz' | tar -x --gzip --strip-components=1 ttf/Hack-Bold.ttf
+          # TODO: Find a better source for Interstate-Bold.ttf
+          curl -sL 'https://github.com/railsgirls/FrozenRailsWeekendHelsinki/raw/62e0e01007a3abc50f4c1c1121b4f05886ac9807/assets/fonts/14.Interstate%20TT%20%20%20%20%20%5B1993%20-%20Tobias%20Frere-Jones%5D/Interstate-Bold.ttf' > Interstate-Bold.ttf
```

Comment:
If you have a better source for this, let me know

Reply:
The OpenCarnage builds includes a text file with the sources of the included fonts

Reply:
Hmm, it says:
```
Interstate-Bold.ttf
    - Page: https://www.azfonts.net/load_font/interst1.html
    - License: "Free for commercial use"
    Note that this font was modified by Vaporeon to fix some errors.
```

That page has a captcha on it so it so I'm not going to be able to download it in in the build script. Plus, I'd need to make the modifications to it as well.

I think I'll just add the fonts to the repo instead.